app/src/pages/ExploreDataPage.tsx

```diff
@@ -26,6 +57,20 @@ function ExploreDataPage() {
 
   return (
     <React.Fragment>
+      <Dialog
+        open={shareModalOpen}
+        onClose={() => setShareModalOpen(false)}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title">Link to this Report</DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+            {window.location.host}/exploredata?mlp={phraseIndex}&
```

Comment:
nit: use MADLIB_PHRASE

As a side note, should we make constants for the main paths so that we don't have to hard-code these (or make a url builder)? Maybe put a TODO in here?

Reply:
Refactored into a util! Also I confused myself with these names, so I appended "param" to the end of the query param consts